Sources/StreamChat/Database/DTOs/ChannelDTO.swift

```diff
@@ -370,7 +370,7 @@ extension _ChatChannel {
             isFrozen: dto.isFrozen,
             lastActiveMembers: { fetchMembers() },
             membership: dto.membership.map { $0.asModel() },
-            currentlyTypingUsers: Set(typingUsers),
+            currentlyTypingUsers: { Set(typingUsers) },
```

Comment:
hm, to keep lazy loading working we should covert user DTOs to models inside the closure, I think 🤔 Otherwise faults are evaluated at the time we create a channel model, not when typing users are requested